pkg/pillar/cmd/zedrouter/metadata.go

```diff
@@ -495,6 +510,54 @@ func (hdl signerHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	w.Write(resp.Bytes())
 }
 
+// ServeHTTP for diagHandler returns text output
+func (hdl diagHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	hdl.zedrouter.log.Tracef("diagHandler.ServeHTTP")
+
+	if r.Method != http.MethodGet {
+		msg := "diagHandler: request method is not GET"
+		hdl.zedrouter.log.Error(msg)
+		http.Error(w, msg, http.StatusMethodNotAllowed)
+		return
+	}
+	remoteIP := net.ParseIP(strings.Split(r.RemoteAddr, ":")[0])
+	anStatus := hdl.zedrouter.lookupAppNetworkStatusByAppIP(remoteIP)
```

Comment:
This code section with `lookupAppNetworkStatusByAppIP` was probably copied from another handler and should not be here.

Reply:
Actually it is there so that in case someone manages to inject a http get to the bridge IP address somehow we check that the source IP is from a local application instance. And the ACLs prevent that source IP from being spoofed.

Reply:
I see, OK

Reply:
I added a comment to the code.

Reply:
I added a comment to the code about that.